sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/users/UserManagementSteps.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.steps.web.application.users;
+
+import static org.sormas.e2etests.pages.application.users.CreateNewUserPage.*;
+import static org.sormas.e2etests.pages.application.users.UserManagementPage.*;
+
+import cucumber.api.java8.En;
+import javax.inject.Inject;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+
+public class UserManagementSteps implements En {
+  protected WebDriverHelpers webDriverHelpers;
+
+  @Inject
+  public UserManagementSteps(WebDriverHelpers webDriverHelpers) {
+    this.webDriverHelpers = webDriverHelpers;
+    When(
+        "^I click on the NEW USER button$",
+        () ->
+            webDriverHelpers.clickWhileOtherButtonIsDisplayed(
+                NEW_USER_BUTTON, FIRST_NAME_OF_USER_INPUT));
+
+    When(
+        "^I select fists user from list$",
+        () -> {
+          selectFirstElementFromList();
+        });
+
+    When(
+        "^I search for created user$",
+        () -> {
+          searchForUser(CreateNewUserSteps.user.getUserName());
+          selectFirstElementFromList();
+        });
+
+    When(
+        "^I search for edited user$",
```

Comment:
let's rename it recently edited user to be more specific

Reply:
changed